ast/functions.go

```diff
@@ -223,6 +223,7 @@ const (
 	SubstringIndex  = "substring_index"
 	ToBase64        = "to_base64"
 	Trim            = "trim"
+	Translate       = "translate"
```

Comment:
Should we add "translate" in UnReservedKeyWord?

Reply:
Yes. `translate` is unreserved keyword.